docs/src/extrapolation_methods.md

```diff
@@ -0,0 +1,56 @@
+# Extrapolation methods
+
+We will use the following interpolation to demonstrate the various extrapolation methods.
+
+```@example tutorial
+using DataInterpolations, Plots
+
+u = [0.86, 0.65, 0.44, 0.76, 0.73]
+t = [0.0022, 0.68, 1.41, 2.22, 2.46]
+t_eval_down = range(-1, first(t), length = 25)
+t_eval_up = range(last(t), 3.5, length = 25)
+A = QuadraticSpline(u, t)
+plot(A)
+```
+
+Extrapolation behavior can be set for `t` beyond the data in the negative and positive direction separately with the `extrapolation_down` and `extrapolation_up` keywords of the interpolation constructors respectively.
```

Comment:
Why do we need `extrapolation_down` or `extrapolation_up`? Doesn't it depend on the data and the last polynomial whether it will go up or down?

Reply:
Ah, apparently these names are not very informative. By down I mean `t < A.t[1]` and by up `t > A.t[end]`. I want to be able to set different extrapolation types on either side.

Reply:
Maybe left and right is better?

Reply:
Backward forward?

Reply:
I'd like to have the option for my application. We could also allow setting  both at once and have the option to set them individually

Reply:
Alright, then we can have that option. We can have `extrapolate` to set the same option on both sides and if users want, they can set different options using `extrapolate_left` or `extrapolate_right`. How does that sound?